apps/chat-e2e/src/tests/chatBarFolderConversation.test.ts

```diff
@@ -685,3 +686,263 @@ dialTest(
+      },
+    );
+
+    await dialTest.step(
+      'Context menu appears if to click on chat name or folder name using right mouse button',
+      async () => {
+        // Chat context menu
+        await conversations
+          .getEntityByName(firstConversation.name)
+          .click({ button: 'right' });
+        await conversations
+          .getDropdownMenu()
+          .waitForState({ state: 'visible' });
+        await conversations.getConversationName(firstConversation.name).click();
+
+        // Folder context menu
+        await folderConversations
+          .getFolderByName(ExpectedConstants.newFolderWithIndexTitle(1))
+          .click({ button: 'right' });
+        await folderConversations
+          .getDropdownMenu()
+          .waitForState({ state: 'visible' });
+      },
+    );
+  },
+);
+
+dialTest(
+  'Menu for conversation in Replay mode\n' +
+    'Single chat can not be stored in Pinned chats section',
+  async ({
+    dialHomePage,
+    conversationData,
+    dataInjector,
+    conversations,
+    conversationDropdownMenu,
+    setTestIds,
+    localStorageManager,
+    conversationAssertion,
+    chatBar,
+    chatBarFolderAssertion,
+  }) => {
+    setTestIds('EPMRTC-1138', 'EPMRTC-1598');
+    const conversation = conversationData.prepareDefaultConversation();
+    await dataInjector.createConversations([conversation]);
+    await localStorageManager.setSelectedConversation(conversation);
+
+    await dialTest.step(
+      'Create New conversation and send any message there',
+      async () => {
+        await dialHomePage.openHomePage();
+        await dialHomePage.waitForPageLoaded();
+      },
+    );
+
+    await dialTest.step(
+      'Click on Replay item in the menu for the chat',
+      async () => {
+        await conversations.openEntityDropdownMenu(conversation.name);
+        await conversationDropdownMenu.selectMenuOption(MenuOptions.replay);
+        await conversationAssertion.assertEntityState(
+          { name: ExpectedConstants.replayConversation + conversation.name },
+          'visible',
+        );
+      },
+    );
+
+    await dialTest.step('Open menu for the conversation', async () => {
+      await conversations.openEntityDropdownMenu(
+        ExpectedConstants.replayConversation + conversation.name,
+      );
+      const expectedMenuOptions = [
+        MenuOptions.select,
+        MenuOptions.rename,
+        MenuOptions.duplicate,
+        MenuOptions.export,
+        MenuOptions.moveTo,
+        MenuOptions.delete,
+      ];
+      const actualMenuOptions =
+        await conversationDropdownMenu.getAllMenuOptions();
+      expect
+        .soft(actualMenuOptions, ExpectedMessages.contextMenuOptionsValid)
+        .toEqual(expectedMenuOptions);
+
+      const excludedMenuOptions = [
+        MenuOptions.playback,
+        MenuOptions.replay,
+        MenuOptions.share,
+        MenuOptions.publish,
+      ];
+      for (const excludedOption of excludedMenuOptions) {
```

Comment:
there is `await conversationDropdownMenuAssertion.assertMenuOptions()` to verify menu options

Reply:
done